projects/ui/src/lib/services/po-notification/po-notification.interface.ts

```diff
@@ -33,6 +33,6 @@ export interface PoNotification {
    */
   orientation?: PoToasterOrientation;
 
-  /** Define em milissegundos o tempo de duração que a notificação ficará disponível em tela. O padrão é 10000 milissegundos. */
+  /** Define em milissegundos o tempo de duração que a notificação ficará disponível em tela. O padrão é 9000 milissegundos. */
```

Comment:
incluir na documentação do notification no portal essa informação, inclusive que se for warning fica fixo.